examples/CMakeLists.txt

```diff
@@ -76,3 +76,10 @@ add_executable(PR_CC pr_cc_attrlist.cpp)
 target_link_libraries(PR_CC ${husky})
 target_link_libraries(PR_CC ${EXTERNAL_LIB})
 set_property(TARGET PR_CC PROPERTY CXX_STANDARD 14)
+
+# TEST
```

Comment:
Yes, I made a mistake there. Is there another way to do this other than template?